app/javascript/styles/doctors.css

```diff
@@ -0,0 +1,91 @@
+/* stylelint-disable csstree/validator */
+.doc-content p {
+  margin-bottom: 0;
+  font-size: 0.9rem;
+}
+
+.doc-content .name {
+  font-size: 0.8rem;
+  font-weight: 600;
+}
+
+/* .doc-content .contact {
+  font-size: 0.7rem;
+} */
```

Comment:
- [ ] Kindly, remove the commented-out code present in this file in order to ensure the repo's cleanliness. 👍